src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3772,7 +3772,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         Title="$(Satellite_Title)"
         ToolPath="$(AlToolPath)"
         ToolExe ="$(AlToolExe)"
-        SdkToolsPath="$(SdkToolsPathMaybeWithx64Architecture)"
+        SdkToolsPath="$(_ALExeToolPath)"
```

Comment:
Will this break any workarounds that we've suggested to customers? Should we continue to honor `SdkToolsPathMaybeWithx64Architecture` if the user sets it? Behind a changewave?

Reply:
If a customer does have a workaround, it simply won't apply anymore. The workaround basically sets `SdkToolsPathMaybeWithx64Architecture` to exactly what `_ALExeToolPath` will be set to. So the worst case is that the customer workaround is an extra target that doesn't need to run.

Reply:
After updating to VS 16.11.0 our build produced hundreds of MSB3084 Warnings, saying the task attempted to find "al.exe" at two location (naming two locations that are literally the same).
Removing the common workaround for AL1703 Warnings did NOT fix this issue, but reverting this commit did it!

Reply:
Could you provide [binary logs](aka.ms/msbuild/binlog) of your issue before/after the fix?

Reply:
Sorry, our company policies do not allow that. Unfortunately I also could not reproduce this with a small solution. Our Original solution contains resources for multiple languages, some third party libraries (like Boot and Eigen) and several C++ projects that are build with the VS 2013 toolset.
This is the full Warning (one per each localized resource file):
`C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(3747,5): warning MSB3084: Task attempted to find "al.exe" in two locations. 1) Under the "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\x64\" processor specific directory which is generated based on SdkToolsPath 2) The x86 specific directory under "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\x64\" which is specified by the SDKToolsPath property. You may be able to solve the problem by doing one of the following:  1) Set the "SDKToolsPath" property to the location of the Microsoft Windows SDK.`
2) I'm still wondering what might have been the second hint to solve the issue...